src/main/java/com/fasterxml/jackson/databind/deser/impl/UnwrappedPropertyHandler.java

```diff
@@ -15,49 +18,93 @@
  * deserializing "unwrapped" values; sets of properties that are
  * embedded (inlined) as properties of parent JSON object.
  */
-public class UnwrappedPropertyHandler
-{
+public class UnwrappedPropertyHandler {
+    /**
+     * We need a placeholder for creator properties that don't have name
+     * but are marked with `@JsonWrapped` annotation.
+     */
+    public final static PropertyName UNWRAPPED_CREATOR_PARAM_NAME = new PropertyName("@JsonUnwrapped");
+
+    protected final List<SettableBeanProperty> _creatorProperties;
     protected final List<SettableBeanProperty> _properties;
+    protected final Set<String> _unwrappedPropertyNames;
```

Comment:
... and if `isUnwrapped()` is not actually needed, can drop this `Set` and code that maintains it.

Reply:
Removed the set and the check!